kythe/cxx/indexer/cxx/GraphObserver.h

```diff
@@ -668,6 +668,9 @@ class GraphObserver {
       const Range& SourceRange, const NodeId& DeclId,
       const absl::optional<NodeId>& DefnId = absl::nullopt) {}
 
+  /// \brief Should an anchor be stamped
+  enum class Stamped { No, Yes };
```

Comment:
nit: prefer to follow the conventions elsewhere and go with something like:

enum class Stamping {
 Stamped, Unstamped
};

rather than just "yes/no"